ps_emailalerts.php

```diff
@@ -751,11 +751,8 @@ public function hookActionProductCoverage($params)
 
     public function hookDisplayHeader()
     {
-        $this->page_name = Dispatcher::getInstance()->getController();
-        if (in_array($this->page_name, ['product', 'account'])) {
-            $this->context->controller->addJS($this->_path . 'js/mailalerts.js');
-            $this->context->controller->addCSS($this->_path . 'css/mailalerts.css', 'all');
-        }
+        $this->context->controller->addJS($this->_path . 'js/mailalerts.js');
```

Comment:
@daresh could you please use `registerStylesheet` and a similar method for the JS? 👍 and the absolutely the best would be to have `actionFrontControllerSetMedia` 💪 

We want to improve the codebase every time there's a situation where we need to add or modify something. https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html

If you need any help with git or something let me know 👍

Reply:
Indeed that would be great, I'll be on it.

Reply:
I have a question: improving this codebase requires me to add a new hook, to register it I must create a new upgrade file and set a version number for the module. Can I do it (assign a new version number)?

Reply:
> I have a question: improving this codebase requires me to add a new hook, to register it I must create a new upgrade file and set a version number for the module. Can I do it (assign a new version number)?

@daresh sure you can 👍